manager/controllers/app/select_modules.go

```diff
@@ -78,9 +88,19 @@ func (p *PathBuilder) FindPaths() []Solution {
 	}
 	// get valid solutions by extending data paths with transformations and selecting an appropriate cluster for each capability
 	solutions = p.validSolutions(solutions)
+	// set empty data format in sink nodes from the chosen modules data format
+	p.setSinkDataFormatFromModule(solutions)
+
 	return solutions
 }
 
+// set empty data formats in sink nodes with the modules data format
```

Comment:
This is relevant for new assets only. Why not deal with format as well in handleNewAsset()?

Reply:
Fixed. Thanks